src/id128/id128.c

```diff
@@ -63,6 +66,112 @@ static int verb_invocation_id(int argc, char **argv, void *userdata) {
         return id128_pretty_print(id, arg_mode);
 }
 
+typedef struct gpt_table_entry {
+        const char *name;
+        const char *cname;
```

Comment:
why show cname? that's internal to our code, and not exported, ehnce not really subject for display?

Reply:
For the pretty printer. It's not shown in the table.